### DISCUSSION TITLE
How to decipher secret in django

### DISCUSSION THREAD

**Discussion by rstambach1** [Onemind-Services-LLC/netbox-secrets#99] [Q&A]
Using netbox-secrets plugin how can I decipher plaintext in django. If I have an instance of a secret object is there a function to call to decipher the secret?

**Comment by abhi1693** [comment]
You can use https://github.com/Onemind-Services-LLC/netbox-secrets/blob/master/netbox_secrets/api/views.py#L109 for reference
